Library/ShareContext.swift

```diff
@@ -62,3 +65,25 @@ public enum ShareContext {
     }
   }
 }
+
+extension ShareContext: Equatable {
+}
+
+public func == (lhs: ShareContext, rhs: ShareContext) -> Bool {
+  switch (lhs, rhs) {
+  case let (.creatorDashboard(lhs), .creatorDashboard(rhs)):
+    return lhs == rhs
+  case let (.discovery(lhs), .discovery(rhs)):
+    return lhs == rhs
+  case let (.liveStream(lhs), .liveStream(rhs)):
+    return lhs == rhs
+  case let (.project(lhs), .project(rhs)):
+    return lhs == rhs
+  case let (.thanks(lhs), .thanks(rhs)):
+    return lhs == rhs
+  case let (.update(lhs), .update(rhs)):
+    return lhs == rhs
+  default:
```

Comment:
is it possible to get rid of the default? we wanna shy away from `default`s so that this will trigger a compile error when new cases are added to the enum